server/routes/createGoal/createGoalController.originalBehaviour.test.ts

```diff
@@ -0,0 +1,506 @@
+import createError from 'http-errors'
+import { Request, Response, NextFunction } from 'express'
+import { SessionData } from 'express-session'
+import type { CreateGoalForm } from 'forms'
+import type { NewGoal } from 'compositeForms'
+import moment from 'moment'
+import validateAddStepForm from './addStepFormValidator'
+import validateCreateGoalForm from './createGoalFormValidator'
+import EducationAndWorkPlanService from '../../services/educationAndWorkPlanService'
+import { anotherValidAddStepForm, aValidAddStepForm } from '../../testsupport/addStepFormTestDataBuilder'
+import { aValidCreateGoalForm } from '../../testsupport/createGoalFormTestDataBuilder'
+import aValidAddNoteForm from '../../testsupport/addNoteFormTestDataBuilder'
+import { aValidCreateGoalDtoWithOneStep } from '../../testsupport/createGoalDtoTestDataBuilder'
+import { toCreateGoalDto } from './mappers/createGoalFormToCreateGoalDtoMapper'
+import aValidPrisonerSummary from '../../testsupport/prisonerSummaryTestDataBuilder'
+import futureGoalTargetDateCalculator from '../futureGoalTargetDateCalculator'
+import CreateGoalControllerOriginalBehaviour from './createGoalController.originalBehaviour'
+
+jest.mock('./addStepFormValidator')
+jest.mock('./createGoalFormValidator')
+jest.mock('./mappers/createGoalFormToCreateGoalDtoMapper')
+
+/**
```

Comment:
This new test class is a verbatim copy and paste of the original `createGoalController.test.ts` but with this comment block. Functionally it's identical to the original.